Cargo.toml

```diff
@@ -6,4 +6,4 @@ authors = ["Ales Katona <almindor@gmail.com>"]
 [dependencies]
 dotenv = "0.12.0"
 postgres = "0.15.2"
-web3 = "0.3.0"
+web3 = { git = "https://github.com/akropolisio/rust-web3" }
```

Comment:
Is this some community fork? I need to know why the switch.

Reply:
When using published crate, It builds fails.
I fixed it https://github.com/akropolisio/rust-web3/commit/b99a4190b246ea50735be3f633d03a12b18a6c62#diff-8e3870b5d7879691f4a03fe1f17ad3b5R35